screens/Map.js

```diff
@@ -95,22 +102,25 @@ export default class Map extends React.Component {
         showsUserLocation={
           this.state.locationResult === "denied" ? false : true
         }
-        showsMyLocationButton={true}
+        showsMyLocationButton={false}
         zoomEnabled={true}
         initialRegion={this.state.initialRegion}
+        region={this.state.region}
       >
         {this.props.markers.map(marker => (
           <MapView.Marker
             key={marker.name}
             coordinate={marker.coordinate}
             title={marker.name}
-            onPress={() => this.handlePress(marker)}
+            onPress={(e) => this.handlePress(e, marker)}
```

Comment:
Passing the event into handlePress now so we can grab the location of the click

Reply:
can we change the name of this method to be more descriptive of what it does? like `setRegion` or `selectMarker`